### ISSUE TITLE
Support for underline syntax

### ISSUE THREAD

**Issue by kaushalmodi** [chaseadamsio/goorgeous#9]
Can you please add support for the underline syntax.

Here's a MWE with various formatting markups:

```
#+TITLE: My Hugo Blog Post in Org
#+DATE: 2017-02-13
#+DRAFT: true

* Formatting
** Fonts
| Format           | Org mode markup syntax |
| *Bold*           | =*Bold*=               |
| /Italics/        | =/Italics/=            |
| _Underline_      | =_Underline_=          |
| =Verbatim=       | ==Verbatim==           |
| +Strike-through+ | =+Strike-through+=     |
```

**Comment by kaushalmodi** [comment]
Above also reveals an issue with another formatting parsing.. `==Verbatim==` should show `=Verbatim=` in monospace. But it is not at the moment.

In emacs, you can run `C-c C-e h o` on the above MWE to see the formatting done by org.

**Comment by chaseadamsio** [comment]
I've been thinking about underline, as it's not supported by blackfriday, the only way I could think to support this is to add an element with a class of "underline" or a style="text-decoration:underline;" which feels kind of messy, but is the only way to handle it. Both are trivial to implement, it's just a matter of deciding the right way to do it.

**Comment by chaseadamsio** [comment]
I'm going to open a separate issue for the verbatim bug. This one should now be fixed by #13.

**Comment by kaushalmodi** [comment]
I confirm the fix :+1:
